docs/refarch/boundedContexts/thirdPartyApi/index.md

```diff
@@ -186,68 +188,111 @@ Once the Verification Request has been fulfilled, the 3rd-Party API BC sends an
 ![Use Case - Example REPLACE ME](./assets/3PAT_PISPBulkTransfer_20210909.png)
 >
 
-### Perform Transfer (universal mode)
```

Comment:
Please add a head section before each `Flow Diagram` which includes a description, i.e. probably the same as the image title box.